decidim-core/app/cells/decidim/author_cell.rb

```diff
@@ -92,7 +92,7 @@ def author_classes
     end
 
     def actionable?
-      return false if options[:has_actions] == false
+      return options[:has_actions] if options.has_key?(:has_actions)
```

Comment:
What about this?

```ruby
options[:has_actions].presence || withdrawable? || flaggable?
```

Reply:
This can return true even if options[:has_actions] is false,  it brokes proposals show view for example because of cell_options: options.merge(has_actions: false) @ decidim-core/app/cells/decidim/collapsible_authors/show.erb.